### PR TITLE
Refactor code to reduce multi-hop commands

### PR DESCRIPTION
**Pull request by Magmanat**

### CODE REVIEW

**Review by typingpanda — DISMISSED** [review]
LGTM!

File: src/main/java/seedu/bigpp/command/commoncommand/BackCommand.java [hunk 1]
```diff
@@ -9,20 +9,13 @@ public class BackCommand extends Command {
...
-        case BUILDER:
-            UI.setViewerMode();
-            return "returned to viewer";
-        case TUTORIAL:
-            UI.setMainMenuMode();
-            return "returned to main menu";
-        case COMPONENT:
-            UI.setBuilderMode();
-            return "returned to builder";
+        case PCVIEWER:
```
Comment (Ryujikjs) [inline src/main/java/seedu/bigpp/command/commoncommand/BackCommand.java:12]: Good simplification of menus

File: src/main/java/seedu/bigpp/ui/UI.java [hunk 2]
```diff
@@ -26,58 +27,44 @@ public static String getInput() {
...
-            break;
-        case VIEWER:
-            PCViewerMenu.printMenu();
-            break;
-        case BUILDER:
-            BuilderMenu.printMenu();
-            break;
-        case TUTORIAL:
-            TutorialMenu.printMenu();
+        case PCVIEWER:
```
Comment (Ryujikjs) [inline src/main/java/seedu/bigpp/ui/UI.java:40]: Good for our application where we need to have long global commands

**Review by Ryujikjs — APPROVED** [review]
Code looks good and passes all gradle checks

**Pull request merged**